src/components/TextField/TextField.tsx

```diff
@@ -7,6 +7,7 @@ type Props = {
   label?: string,
   required?: boolean,
   onChange?: (newValue: string) => void,
+  isValid?: (newValue: string) => boolean,
```

Comment:
```suggestion
  validate?: (newValue: string) => boolean,
```

Name it as a function